### PR TITLE
feat: Add new demo content

### PR DESCRIPTION
**Pull request by podarok**
WiP: https://github.com/YCloudYUSA/y_lb_demo_content/pull/7

Original Issue, this PR is going to fix: REPLACE WITH A LINK TO ISSUE ( publicly available )

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

 - go to admin/modules
 - enable the Y Layout Builder - Init Demo Pages module
 - go to /admin/structure/migrate/manage/y_lb_init_demo_content/migrations
 - run this migration /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_media_image/execute
 - run these migrations : 
    -  /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_homepage/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_who_we_are/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_programs_overview/execute
    - /admin/structure/migrate/manage/y_lb_init_demo_content/migrations/y_lb_init_demo_page_programs_category/execute
 -  as alternative you can run this drush command: `drush migrate:import --group y_lb_init_demo_content`
 - go to /admin/content
 - visit all page with prefix in the title " Init Demo"
 - verify that pages looks like on design

- [ ] Check http://profile.openy.cibox.tools/build4428/demo-layout-builder-page-home
image

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by AnastasiiaPys** [comment]
retest this please

File: openy.packages.yml [hunk 1]
```diff
@@ -372,6 +372,7 @@ demo:
     - openy_demo_tfitness
     - openy_demo_tnews
     - openy_demo_webform
+    - y_lb_demo_content
```
Comment (aleevas) [inline openy.packages.yml:375]: @podarok please add here a new submodule for the new demo content`  y_lb_init_demo_content`

File: openy.profile [hunk 2]
```diff
@@ -180,6 +180,7 @@ function openy_demo_content_configs_map($key = NULL) {
       'openy_demo_bsimple',
       'openy_demo_bamenities',
       'openy_demo_tfitness',
+      'y_lb_demo_content'
```
Comment (aleevas) [inline openy.profile:183]: @podarok please replace this migration group with the a new one `y_lb_init_demo_content`

File: openy.packages.yml (hunk 1 above)
Comment (podarok) [reply openy.packages.yml:375]: done

File: openy.profile (hunk 2 above)
Comment (podarok) [reply openy.profile]: done

File: openy.profile [hunk 3]
```diff
@@ -180,6 +180,7 @@ function openy_demo_content_configs_map($key = NULL) {
       'openy_demo_bsimple',
       'openy_demo_bamenities',
       'openy_demo_tfitness',
+      'y_lb_init_demo_content'
```
Comment (aleevas) [inline openy.profile:183]: @podarok this one needs to be replaced with a new migration group `y_lb_init_demo_content`

Comment (podarok) [reply openy.profile:183]: ))

image


@aleevas

Comment (aleevas) [reply openy.profile:183]: @podarok sorry my bad, its already ok

File: composer.json [hunk 4]
```diff
@@ -158,7 +158,7 @@
         "symfony/dom-crawler": "^6.2",
         "ycloudyusa/useless_machines": "dev-main",
         "ycloudyusa/y_lb": ">=2.5.1, <2.6.0",
-        "ycloudyusa/y_lb_demo_content": "^1 || dev-main",
+        "ycloudyusa/y_lb_demo_content": "v2.x-dev",
```
Comment (froboy) [inline composer.json:161]: `v2` is not going to be the final branch name, right?

**Review by froboy — APPROVED** [review]
It looks like there's lots of QA coming in, but in general this stuff looks great!

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by mkdok** [comment]
retest this please

**Comment by mkdok** [comment]
retest this please

**Comment by mkdok** [comment]
retest this please

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by mkdok** [comment]
retest this please

**Comment by podarok** [comment]
retest this please

**Comment by mkdok** [comment]
retest this please

**Comment by mkdok** [comment]
retest this please

**Comment by aleevas** [comment]
retest this, please

**Comment by mkdok** [comment]
retest this please

**Comment by froboy** [comment]
retest this please

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by mkdok** [comment]
retest this please

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by froboy** [comment]
retest this please

**Comment by AnastasiiaPys** [comment]
retest this please

**Comment by AnastasiiaPys** [comment]
retest this please

File: composer.json (hunk 4 above)
Comment (podarok) [reply composer.json:161]: sure, it's going to be release tag with ^Caret @froboy

**Pull request closed without merging**